File: my-fragment/src/main/java/com/laioffer/my_fragment/demo1_静态fragment/Demo1Activity.java
package com.laioffer.my_fragment.demo1_静态fragment;

import android.os.Bundle;
import android.view.Window;

import androidx.appcompat.app.AppCompatActivity;

import com.laioffer.my_fragment.R;
import com.laioffer.my_fragment.common.SimpleContentFragment;
import com.laioffer.my_fragment.common.TitleFragment;

/**
 * demo1:静态fragment的使用
 */
public class Demo1Activity extends AppCompatActivity {

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    supportRequestWindowFeature(Window.FEATURE_NO_TITLE);
    setContentView(R.layout.activity_demo1);

    configTitleFragment();
  }

  private void configTitleFragment() {
    TitleFragment titleFragment =
        (TitleFragment) getSupportFragmentManager().findFragmentById(R.id.title_fragment);
    SimpleContentFragment contentFragment =
        (SimpleContentFragment) getSupportFragmentManager().findFragmentById(
            R.id.simple_content_fragment);

    titleFragment.setTitle("[静态fragment]");
    contentFragment.setContent("demo1 Content Fragment");
  }
}

File: my-mvx/src/main/java/com/laioffer/my_mvx/less1_Lifecycle/demo1_原始写法/Demo1Activity.kt
package com.laioffer.my_mvx.less1_Lifecycle.demo1_原始写法

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.laioffer.my_mvx.R

/**
 * @description: 版本1: 监听器监听生命周期
 * @date: 2022/9/19 7:52 上午
 */
class Demo1Activity : AppCompatActivity() {

  private var myListener: MyListener? = null

  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.empty_layout)

    myListener = MyListener()
  }

  override fun onStart() {
    super.onStart()
    myListener?.start()
  }

  override fun onStop() {
    super.onStop()
    myListener?.stop()
  }
}